tests/testthat/test-helper_functions.R

```diff
@@ -0,0 +1,14 @@
+test_that("Successful connection message", {
```

Comment:
If you wanted to complete the set you could check some others like 399 or 20003 return NULL?

Reply:
Prompted some extra work on `http_request_error()` to make it a bit better and added the tests.

Reply:
Nice!